protocols/migration/migration_reports_4_0_0_to_reports_5_0_0.py

```diff
@@ -476,7 +482,7 @@ def migrate_reported_variant_cancer(self, old_instance, assembly, participant_id
             new_instance.alleleFrequencies = [reports_5_0_0.AlleleFrequency(
                 study='genomics_england',
                 population='ALL',
-                alternateFrequency=float(old_instance.reportedVariantCancer.commonAf)
+                alternateFrequency=self.convert_string_to_float(old_instance.reportedVariantCancer.commonAf)
```

Comment:
FYI @priesgo `commonAF` here is an integer:
https://github.com/genomicsengland/GelReportModels/blob/8eefde23e6ff8227bafcf25353cb6b200cde889f/schemas/IDLs/org.gel.models.report.avro/4.0.0/CommonInterpretedCancer.avdl#L250
but fortunately the `convert_string_to_float` method won't fail but we should probably look at this

Reply:
yes, can you correct that?

Reply:
So previously @priesgo, it just did `float(old_instance.reportedVariantCancer.commonAf)` which I think is reasonable.

Are you happy for it go back to that ?